test/language/expressions/optional-chaining/call-expression-optional-chain.js

```diff
@@ -0,0 +1,17 @@
+/*---
+esid: pending
```

Comment:
please use `esid: prod-OptionalExpression` instead of any pending in the test files.